tests/kernel/msgq/msgq_api/src/test_msgq_contexts.c

```diff
@@ -88,6 +88,48 @@ static void msgq_thread(struct k_msgq *pmsgq)
 	purge_msgq(pmsgq);
 }
 
+static void thread_entry_overflow(void *p1, void *p2, void *p3)
+{
+	int ret;
+
+	u32_t rx_buf[MSGQ_LEN];
+
+	ret = k_msgq_get(p1, &rx_buf[0], K_FOREVER);
+
+	zassert_false(ret, NULL);
+
+	ret = k_msgq_get(p1, &rx_buf[1], K_FOREVER);
+
+	zassert_false(ret, NULL);
+
+	k_sem_give(&end_sema);
+}
+
+static void msgq_thread_overflow(struct k_msgq *pmsgq)
+{
+	int ret;
+
+	ret = k_msgq_put(pmsgq, (void *)&data[0], K_FOREVER);
+
+	zassert_false(ret, NULL);
```

Comment:
please check for the actual return value you are expecting, with zassert_equal, not with zassert_true/false

Reply:
Done